packages/vsx-registry/src/node/vsx-extension-resolver.ts

```diff
@@ -98,6 +106,7 @@ export class VSXExtensionResolver implements PluginDeployerResolver {
     }
 
     protected async download(downloadUrl: string, downloadPath: string): Promise<boolean> {
+        if (fs.existsSync(downloadPath)) { return true; }
```

Comment:
Suggestion: We can perform this check async
```suggestion
        if (await fs.pathExists(downloadPath)) {
            return true;
        }
```